packages/ddp-client/common/livedata_connection.js

```diff
@@ -102,6 +106,7 @@ export class Connection {
 
     self._heartbeatInterval = options.heartbeatInterval;
     self._heartbeatTimeout = options.heartbeatTimeout;
+    self._ignoredMsgsForSessionOutOfDateCheck = ['ping', 'pong'];
```

Comment:
events like 'connected', 'nosub', no matter here?

Reply:
Yes, `connected` and `nosub` are intentionally counted in `receivedCount`. Only `ping/pong` are excluded because they're heartbeat noise.

Counting `connected` and `nosub` is important because:
- `connected`is sent once per connection and matters for session state
- `nosub` indicates subscription failure - losing this could cause client/server desync

If we excluded them, the client and server counts could diverge, forcing unnecessary session resets on reconnection.